scene/main/viewport.cpp

```diff
@@ -3950,7 +3950,7 @@ void SubViewport::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_clear_mode"), &SubViewport::get_clear_mode);
 
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "size"), "set_size", "get_size");
```

Comment:
This also seems to be a `Size2i`, i.e. `VECTOR2I`.

Reply:
Just this one left.

Reply:
Missed this message, working on it!